raft-tech/TANF-app
# fix(1487): upgrade django-admin-508 to fix select all action

## Summary of Changes

This PR fixes the select all action link in certain parts of Django Admin

Pull request closes #1487

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Navigate to Django Admin
2. Navigate to the STT table (or any table with more than 100 objects)
3. Select any number of rows
4. Ensure the select all action appears
5. Click select all action and ensure copy is updated with the number of selected rows
6. Try to perform a bulk delete or another action while there is a bulk selection
7. Ensure the action occurs with the proper selection

![Screen Shot 2022-04-05 at 10 42 18 AM](https://user-images.githubusercontent.com/12901172/161780889-718d6509-d037-4501-9f96-2c7ef023c1ce.png)
![Screen Shot 2022-04-05 at 10 42 29 AM](https://user-images.githubusercontent.com/12901172/161780886-cbeebd34-0edc-45c8-9702-b9fc149263ff.png)
![Screen Shot 2022-04-05 at 10 42 36 AM](https://user-images.githubusercontent.com/12901172/161780885-122ce384-be98-4437-8770-959ac19a8079.png)
![Screen Shot 2022-04-05 at 10 42 46 AM](https://user-images.githubusercontent.com/12901172/161780884-0d6a964e-982d-40e2-b210-80cddbe8e49c.png)
![Screen Shot 2022-04-05 at 10 42 53 AM](https://user-images.githubusercontent.com/12901172/161780881-3d1f0609-154b-47aa-931e-6311f015bd45.png)
![Screen Shot 2022-04-05 at 10 42 59 AM](https://user-images.githubusercontent.com/12901172/161780879-3294ee47-db3e-44b8-b167-9d041499e184.png)


## Tabletop


## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x]  Select all Action is functional

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

_no_


### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
